docs/variants.rst

```diff
@@ -28,18 +43,18 @@ The variants file can be placed in the root of the project directory, or in the
 What does it look like?
 =======================
 
-A simple two-option ``cmake-variants.yaml`` might look like this:
+A simple two-setting ``cmake-variants.yaml`` might look like this:
 
 .. image:: res/variants_yaml.png
     :align: center
 
-This file defines two variant options: *buildType* and *useOpenGL*. They each
-have two settings of their own, defined by the ``choices`` key.
+This file defines two variant **settings**: *buildType* and *useOpenGL*. They
+each have two **options** defined by the ``choices`` key.
 
 In total, the number of possible variants is defined by the cartesian product
-of the possible choices. Two options with two settings each creates *four*
-variants. When we ask to change the build type, CMake Tools will present each
-possible combination in a quickpick:
+of the possible choices. Two **settings** each with two **options** creates
+*four* variants. When we ask to change the build type, CMake Tools will present
```

Comment:
I would introduce variant a a important word. This underlines the meaning of the word and the meaning for later sections.